modules/raft/src/main/java/org/apache/ignite/raft/jraft/JRaftServiceFactory.java

```diff
@@ -31,11 +31,11 @@ public interface JRaftServiceFactory {
     /**
      * Creates a raft log storage.
      *
-     * @param uri The log storage uri from {@link NodeOptions#getSnapshotUri()}
+     * @param groupId The name of the raft group.
      * @param raftOptions the raft options.
      * @return storage to store raft log entires.
      */
-    LogStorage createLogStorage(final String uri, final RaftOptions raftOptions);
+    LogStorage createLogStorage(final String groupId, final RaftOptions raftOptions);
```

Comment:
Everything else works with the URI. Why is this change needed? It doesn't look very consistent.

Reply:
All log storages have the same URI in this PR. The only way to distinguish data in a storage is by raft group id. Hence the change